readthedocs/rtd_tests/tests/test_subprojects.py

```diff
@@ -152,24 +152,34 @@ def setUp(self):
             self.owner = create_user(username='owner', password='test')
             self.tester = create_user(username='tester', password='test')
             self.pip = fixture.get(Project, slug='pip', users=[self.owner], main_language_project=None)
-            self.subproject = fixture.get(Project, slug='sub', language='ja',
-                                          users=[ self.owner],
-                                          main_language_project=None)
-            self.translation = fixture.get(Project, slug='trans', language='ja',
-                                           users=[ self.owner],
-                                           main_language_project=None)
+            self.subproject = fixture.get(Project, slug='sub', language='ja', users=[
+                                  self.owner], main_language_project=None)
+            self.translation = fixture.get(Project, slug='trans', language='ja', users=[
+                                   self.owner], main_language_project=None)
             self.pip.add_subproject(self.subproject)
             self.pip.translations.add(self.translation)
 
-    @override_settings(PRODUCTION_DOMAIN='readthedocs.org')
-    def test_resolver_subproject_alias(self):
         relation = self.pip.subprojects.first()
         relation.alias = 'sub_alias'
         relation.save()
+        fixture.get(Project, slug='sub_alias', language='ya')
+
+
+    @override_settings(PRODUCTION_DOMAIN='readthedocs.org')
+    def test_resolver_subproject_alias(self):
         with override_settings(USE_SUBDOMAIN=False):
             resp = self.client.get('/docs/pip/projects/sub_alias/')
             self.assertEqual(resp.status_code, 302)
             self.assertEqual(
                 resp._headers['location'][1],
                 'http://readthedocs.org/docs/pip/projects/sub_alias/ja/latest/'
             )
+
+    def test_resolver_subproject_subdomain_alias(self):
+        with override_settings(USE_SUBDOMAIN=True):
```

Comment:
Can you use `override_settings` as decorator?

Reply:
done :)